### PR TITLE
Add ACL configuration for USIR group and schema

### PR DESCRIPTION
**Pull request by rimolive**

### CODE REVIEW

File: kfdefs/base/trino/trino-config-secret.yaml [hunk 1]
```diff
@@ -86,8 +86,18 @@ stringData:
...
         },
         {
-          "group": "ccx|grokket|product-marketing|openshift|openshift-community-enablement|acm-observability|openshift-qe|ansible-engineering|usir|openshift-analytics",
-          "schema": "telemetry|jdbc",
+          "group": "ccx|grokket|product-marketing|openshift|openshift-community-enablement|acm-observability|openshift-qe|ansible-engineering|openshift-analytics",
+          "schema": "telemetry",
+          "owner": true
+        },
+        {
+          "group": "ccx|grokket|product-marketing|openshift|openshift-community-enablement|acm-observability|openshift-qe|ansible-engineering|openshift-analytics|usir",
```
Comment (accorvin) [inline kfdefs/base/trino/trino-config-secret.yaml:94]: I wonder if there's a built in group that captures all users that we could replace this with

Comment (rimolive) [reply kfdefs/base/trino/trino-config-secret.yaml:94]: Unfortunately, no. Until we have LDAP Group provider added to Trino we can't have other ways to manage groups and ACL.

**Review by maulikjs — APPROVED** [review]

**Pull request merged**